aten/src/ATen/native/Pow.cpp

```diff
@@ -77,6 +77,70 @@ Tensor pow(Scalar base, const Tensor& exp) {
   return native::pow_out(result, base, exp);
 }
 
+Tensor& float_power_out(Tensor& result, const Tensor& base, const Tensor& exp) {
+  auto dtype = (at::isComplexType(base.scalar_type()) || at::isComplexType(exp.scalar_type())) ?
+                at::kComplexDouble : at::kDouble;
+  TORCH_CHECK(result.scalar_type() == dtype,
+              "output type ", result.scalar_type(), "is not the desired output type ", dtype);
+
+  auto iter = TensorIterator::binary_op(result, base.to(dtype), exp.to(dtype));
+  pow_tensor_tensor_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor& float_power_out(Tensor& result, const Tensor& base, Scalar exp) {
+  // Numpy compatibility check:
+  TORCH_CHECK(!(isIntegralType(base.scalar_type(), true) &&
+              exp.isIntegral(true) && exp.toLong() < 0),
+              "Integers to negative integer powers are not allowed.");
+
+  auto dtype = (at::isComplexType(base.scalar_type()) || exp.isComplex()) ? at::kComplexDouble : at::kDouble;
+  TORCH_CHECK(result.scalar_type() == dtype,
+              "output type ", result.scalar_type(), "is not the desired output type ", dtype);
+
+  if (exp.isComplex() && (exp.toComplexDouble() == 0.0) ) {
+    result.resize_as_(base).fill_(1);
+  } else if (exp.isComplex() && (exp.toComplexDouble() == 1.0) ) {
+    result.resize_as_(base).fill_(base);
```

Comment:
.copy_?

Reply:
Would you please fix this for pow, too, actually. Sample code demonstrating the error: 

```
torch.tensor((1 + 1j, 2 + 2j)) ** complex(1, 0)
: RuntimeError: fill_ only supports 0-dimension value tensor but got tensor with 1 dimensions.
```

Reply:
Progressing as a separate PR #46830